.github/workflows/delete-branch-on-merge-to-master.yml

```diff
@@ -0,0 +1,27 @@
+name: Selective delete feature branch after merge to master
+
+# Only run the workflow when a PR that targets master is closed
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'master'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+jobs:
+  merge_job:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Delete merged branch
+    # from the github actions marketplace repo here: https://github.com/pabio/delete-merged-action
+      uses: koj-co/delete-merged-action@master
+      with:
+      # do not delete master, development, or hot-fix-release branches
+        branches: "!master, !development, !hot-fix*, !hotfix*, *"
```

Comment:
Blocker: SCP conventionally starts all feature branch names with the author's initials.  Of the 64 search [hits for "hotfix" in this repo](https://github.com/broadinstitute/single_cell_portal_core/search?q=hotfix&type=issues), I haven't seen any examples of branch names beginning with "hotfix" in the handful of cases I glanced over.  So this new requirement strikes me as:

* quite feasible overlook,
* cause delays in urgent changes when that happens, and
* not of notable cost to omit

Regarding that last point, in other words, _not_ deleting a branch named e.g. `eh-hotfix-violin-plots` or `jb-registration-hotfix` seems basically fine.  The cost of not deleting it is very low -- some extra cruft among open branches that can be cleaned up in a few clicks.  The cost of deleting the branch unintentionally is notably higher -- it can (IIUC) substantially delay the release of an urgent fix.  

So relaxing the rules to not delete a branch -- e.g. don't delete any branches that have "hotfix" _anywhere_ in the name -- would reduce those expensive false positives (unintended deletes) for a negligible cost in false negatives (unintended keeps).

```suggestion
        branches: "!master, !development, !*hotfix*, !*hot-fix*, *"
```

Reply:
Ah I am realizing I checked PR titles not branch titles when checking for how folks had been doing hotfix branches. Making the check for hotfix in the title rather than beginning with it makes sense to me I will make that update. 

As an aside I am not convinced this is a blocker and would push back on that/using request-changes for this update.

Reply:
Thanks for raising that concern on the "blocker" prefix and "request changes" status.  

My previous comment describes a functional problem that directly relates, which seems to warrant a "Blocker".  The fix itself did seem seem trivial, so I agree "Request changes" wasn't ideal!